vita3k/io/src/io.cpp

```diff
@@ -862,7 +862,7 @@ int remove_dir(IOState &io, const char *dir, const std::wstring &pref_path, cons
     return 0;
 }
 
-static std::string standardize_path(std::string_view path) {
+static std::string standardize_path(const std::string_view path) {
```

Comment:
It's not really needed (not adding a lot of meaning) to make a string_view const